certbot/tests/error_handler_test.py

```diff
@@ -23,7 +25,7 @@ def set_signals(sig_handler_dict):
 def signal_receiver(signums):
     """Context manager to catch signals"""
     signals = []
-    prev_handlers = {}
+    prev_handlers = {}  # type: Dict[int, Union[int, None, Callable[[], None]]]
```

Comment:
See my comment on `error_handler.py` about what I think the type of the callable here should be.

Reply:
yes, fixed